assets/stylesheets/common/discourse-calendar.scss

```
@@ -106,3 +111,31 @@ a.holiday {
 .combo-box.user-timezone {
   min-width: 15em;
 }
+
+.discourse-calendar-header {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+  padding: 0.5em;
+  border-bottom: 1px solid $primary-low;
+  background: $primary-very-low;
+
+  .discourse-calendar-timezone-picker {
+    margin-bottom: 0;
+  }
+
+  .discourse-calendar-title {
+    margin-top: 10px;
```

That's personal preference, but:

```suggestion
    margin-top: 1em;
```